EKrut-Server/src/ekrut/server/managers/ServerInventoryManager.java

```diff
@@ -42,6 +62,14 @@ public InventoryItemResponse updateItemQuantity(InventoryItemRequest inventoryUp
 		return new InventoryItemResponse("OK");
 	}
 	
+	
+	/**
+	 * Handles Client's request to <b>get</b> InventoryItem(s).
+	 * 
+	 * @param inventoryUpdateItemRequest the InventoryItemRequest that was instantiated with:
```

Comment:
The `@param` tag requires specifying the parameter name exactly. Here for example it's called `inventoryGetItemsRequest` so that's what it's supposed to be called here.